src/renderers/dom/server/ReactServerUpdateQueue.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactServerUpdateQueue
+ * @flow
+ */
+
+'use strict';
+
+var ReactUpdateQueue = require('ReactUpdateQueue');
+var Transaction = require('Transaction');
+var ReactClass = require('ReactClass');
+var warning = require('warning');
+
+function warnTDZ(publicInstance: ReactClass, callerName: string) {
```

Comment:
TDZ already has meaning in JS. While it's sort of true that we're warning against a sort of "React TDZ", I think I'd prefer if we didn't overload that terminology.